modules/ROOT/pages/external-org-access.adoc

```diff
@@ -0,0 +1,41 @@
+= External Organization Access
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Organizations that have their own, separate domains in Anypoint Platform can share resources with one another using external access lists of trusted organizations.
+This enables your organization to maintain an independent Anypoint Platform organization without keeping resources isolated from external organizations that have partnerships or shared projects.
+
+As an organization administrator, you can specify which external organizations can share resources with your organization.
+In turn, an external organization that has your organization's unique domain name can add your organization to its list of trusted organizations so that your organization can share assets with it.
+
+When an external organization elects to share resources with your organization, the external organization does not receive any information about your organization.
+Additionally, external organizations must have your organization's unique domain name to share resources.
```

Comment:
This is not true; if you've whitelisted the external org, they will be able to look up your org by name (when sharing in Exchange). I think it's okay to exclude this info altogether and mention it on the Exchange side.

Reply:
```suggestion
```